vital/util/string.cxx

```diff
@@ -95,4 +96,31 @@ join( const std::vector<std::string>& elements, const std::string& str_separator
 }
 
 
+//----------------------------------------------------------------------------
+/**
+ * Removes duplicate strings in a vector while preserving original order
+ */
+void
+erase_duplicates(std::vector<std::string>& items)
```

Comment:
Wold there be any value in templating this function rather than just handling just std::string?

Reply:
I'd say no, unless there was some specific need for it in the future.